### PR TITLE
Add parcel delineation to python examples

### PR DESCRIPTION
**Pull request by VictorVerhaert**
The example focuses on the inference step, using pre-trained models. It demonstrates data loading and preprocessing, 
inference, and finally producing vector data as a result.

### CODE REVIEW

File: python/ParcelDelineation/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Tuning by:
+- Kasper Bonte
+- Bart Driessen
+
+[VITO Remote Sensing](https://remotesensing.vito.be)
+
+## Running the example
+
+You will require an openEO account to run the example. The [openEO platform](https://openeo.cloud/) offers various options for this.
```
Comment (jdries) [inline python/ParcelDelineation/README.md:24]: best to remove this sentence about accounts, as this is a platform independent example. Info about how to get accounts is available elsewhere.

File: python/ParcelDelineation/udf_sobel_felzenszwalb.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import xarray
+from openeo.udf import XarrayDataCube
+from skimage import segmentation
+from skimage.filters import sobel
+from skimage import graph
+
+
+def apply_datacube(cube: XarrayDataCube, context: dict) -> XarrayDataCube:
```
Comment (jdries) [inline python/ParcelDelineation/udf_sobel_felzenszwalb.py:9]: like the other UDF, can we also here avoid the use of XarrayDataCube and just use a DataArray?

**Review by jdries — CHANGES_REQUESTED** [review]
In the notebook itself:
- try to make the inrto more generic, no need to reference a specific platform

It mentions tensorflow, but we use onnx, perhaps good to mention that we converted the model to onnx, to have a more standardized model format.

File: python/ParcelDelineation/README.md (hunk 1 above)
Comment (VictorVerhaert) [reply python/ParcelDelineation/README.md]: I have replaced it with a reference to the openEO hub for finding providers.

**Review by jdries — APPROVED** [review]
Approved, I don't see any remaining showstoppers.
I suggest to merge, as users are frequently asking about this, so would be get that we can point to this one.

**Pull request merged**